Update docs to be time-neutral by removing outdated references, update related screenshots to where these issues occured (#396)
actualbudget/docs: docs/budgeting/categories.md

![](/img/categories/CategoriesMerged.png)

## Managing Categories

> Category groups can be *expanded or collapsed* by clicking on the arrow to the left of the Category group name. All category groups can be expanded or collapsed at once through use of the three dot menu at the top of the category list.

Reviewer: ```suggestion
Category groups can be *expanded or collapsed* by clicking on the arrow to the left of the Category group name. All category groups can be expanded or collapsed at once by using the three-dot menu at the top of the category list.
```

> Categories that are not currently needed can be *hidden* by using the down arrow by each category name. An example is a budgeting savings for a large cost(e.g. washing machine). After the purchase is made, the category can be hidden. Hidden categories still impact your budget. When needed, hidden categories can be viewed by clicking on the three dots at the top of the category list. 

Reviewer: ```suggestion
Categories that are not currently needed can be *hidden* by using the down arrow by each category name. An example is budgeting savings for a large cost (e.g. washing machine). After the purchase is made, the category can be hidden. Hidden categories still impact your budget. When needed, hidden categories can be viewed by clicking on the three dots at the top of the category list. 
```

![](/img/categories/CategoryToggleHidden.png)

> Hidden categories, when shown, are displayed in a lower-contrast color.

Reviewer 2: It might be worth highlighting that hidden categories can be unhidden (it should be intuitive, but then again, so should hiding categories)

```suggestion
Hidden categories, when shown, are displayed in a lower-contrast color. If you find that you now require a category that you have previously hidden, you can unhide it by using the down arrow next to the category name.
```

> Hidden categories, when shown, are displayed in a low-contrast color.

Reviewer: Lower, not low. Low contrast implies it might be hard to read, which (hopefully) is not the case.

```suggestion
Hidden categories, when shown, are displayed in a lower-contrast color.
```

![](/img/categories/CategoriesMerged.png)

## Expanding, Collapsing or Hiding Categories

> Category groups can be *expanded or collapsed* by clicking on the three dots at the top of the Category list. Categories that are not currently needed can be *hidden* by using the down arrow by each category name. Hidden categories still impact your budget.

Reviewer 3: _Categories that are not currently needed can be *hidden* by using the down arrow by each category name._ 

I do understand what the text is saying - but for a new user, this sentence makes less sense.

I suggest having 2 paragraphs, one for _expanding / collapsing the category **group**_, and one paragraph dealing with _hiding and unhiding categories within the category **group**_.

Regarding hiding categories, we should have some examples of why one would like to do this. When I teach zero-som budgeting and categories I use "Saving to buy a new washing machine" as an example of a category that should be hidden when the washing machine is bought.

Author: i like the idea behind providing an example, can incorporate into this pr

Author: could use some help with the grammar i think but

> Hidden categories can be viewed, when needed, by clicking on the three dots at the top of the category list. 

Reviewer 3: To simplify the language, I suggest a variant of _When needed, hidden categories can be viewed by clicking on the three dots at the top of the category list. _
